src/main/handlers/misc/saveChangesDialog.ts

```diff
@@ -0,0 +1,27 @@
+import { BrowserWindow, dialog } from 'electron';
```

Comment:
This handler should probably be under `handlers/file` instead of `handlers/misc`